src/main/java/bstore/bookstore/service/BookServiceImpl.java

```diff
@@ -35,4 +40,22 @@ public List<BookDto> findAll() {
                 .map(bookMapper::toDto)
                 .toList();
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public BookDto updateById(Long id, CreateBookRequestDto requestDto) {
+        Book book = bookRepository.findById(id)
+                .orElseThrow(() -> new EntityNotFoundException("Book not found with id: " + id));
+        book.setTitle(requestDto.getTitle());
+        book.setAuthor(requestDto.getAuthor());
+        book.setIsbn(requestDto.getIsbn());
+        book.setPrice(requestDto.getPrice());
+        book.setDescription(requestDto.getDescription());
+        book.setCoverImage(requestDto.getCoverImage());
```

Comment:
use mapper for it. Create method `Book updateBook(Book book, CreateBookRequestDto requestDto)`